app/models/export_template.rb

```diff
@@ -85,8 +170,76 @@ def tags_for_pj
+      entreprise_nom: { libelle: 'Entreprise nom', get_value: -> (d) { d.etablissement&.entreprise_nom } },
+      entreprise_prenom: { libelle: 'Entreprise prénom', get_value: -> (d) { d.etablissement&.entreprise_prenom } },
+      association_rna: { libelle: 'Association RNA', get_value: -> (d) { d.etablissement&.association_rna } },
+      association_titre: { libelle: 'Association titre', get_value: -> (d) { d.etablissement&.association_titre } },
+      association_objet: { libelle: 'Association objet', get_value: -> (d) { d.etablissement&.association_objet } },
+      association_date_creation: { libelle: 'Association date de création', get_value: -> (d) { d.etablissement&.association_date_creation } },
+      association_date_declaration: { libelle: 'Association date de déclaration', get_value: -> (d) { d.etablissement&.association_date_declaration } },
+      association_date_publication: { libelle: 'Association date de publication', get_value: -> (d) { d.etablissement&.association_date_publication } },
+      domaine_fonctionnel: { libelle: 'Domaine Fonctionnel', get_value: -> (d) { d.procedure.chorus_configuration.domaine_fonctionnel&.fetch("code") { '' } } },
+      referentiel_prog: { libelle: 'Référentiel De Programmation', get_value: -> (d) { d.procedure.chorus_configuration.referentiel_de_programmation&.fetch("code") { '' } } },
+      centre_de_cout: { libelle: 'Centre De Coût', get_value: -> (d) { d.procedure.chorus_configuration.centre_de_cout&.fetch("code") { '' } } },
+      archived: { libelle: 'Archivé', get_value: -> (d) { d.archived } },
+      dossier_state: { libelle: 'État du dossier', get_value: -> (d) { Dossier.human_attribute_name("state.#{d.state}") } },
+      updated_at: { libelle: 'Dernière mise à jour le', get_value: -> (d) { d.updated_at } },
+      last_champ_updated_at: { libelle: 'Dernière mise à jour du dossier le', get_value: -> (d) { d.last_champ_updated_at } },
+      depose_at: { libelle: 'Déposé le', get_value: -> (d) { d.depose_at } },
+      en_instruction_at: { libelle: 'Passé en instruction le', get_value: -> (d) { d.en_instruction_at } },
+      sva_svr_decision_on: { libelle: "Date décision #{procedure.sva_svr_configuration.human_decision}", get_value: -> (d) { d.sva_svr_decision_on } },
+      processed_at: { libelle: 'Traité le', get_value: -> (d) { d.processed_at } },
+      motivation: { libelle: 'Motivation de la décision', get_value: -> (d) { d.motivation } },
+      instructeurs: { libelle: 'Instructeurs', get_value: -> (d) { d.followers_instructeurs.map(&:email).join(' ') } },
+      groupe_instructeur: { libelle: 'Groupe instructeur', get_value: -> (d) { d.groupe_instructeur.label } }
+    }
+  end
+
   private
 
+  def libelle_for_path(full_path, active_libelle)
+    columns&.find { _1["path"] == full_path && _1["stable_id"] == full_path[:stable_id] }&.fetch("libelle", nil) || active_libelle
+  end
```

Comment:
- l'implem semble fausse car : 
  - `_1["path"] == full_path` : full_path est un string,
  -  `_1["stable_id"] == full_path[:stable_id]`  full_path est un hash

- je te propose de changer l'implem pour que la méthode renvoie uniquement un libelle sauvé et que sa signature prenne un path de type hash

```suggestion
  def saved_libelle(path_h)
    columns&.find { _1.slice('path', 'stable_id') == path_h.slice('path', 'stable_id') }&.dig("libelle")
  end
```

- on laisse a l'appelant le soin de faire un `libelle = saved_libelle(path_h) || active_libelle`

- par ailleurs, j'ai l'impression que cette methode ne marche pas pour les columns qui n'ont pas de stable_id (celles qui concernent le dossier par exemple)

Reply:
Ce commit utilise `saved_libelle` : https://github.com/demarches-simplifiees/demarches-simplifiees.fr/pull/10591/commits/4318980ce3a048014e365dedf0eb8b0ecb995fbf

Reply:
pour les colonnes qui n'ont pas de stable_id (comme celles conernant le dossier), la méthode `saved_libelle` n'est pas appelée. On en a besoin uniquement pour les champs qui sont renommés lors d'une nouvelle révision